src/PlaywrightSharp.Chromium/ChromiumNetworkManager.cs

```diff
@@ -44,6 +46,12 @@ internal Task SetCacheEnabledAsync(bool enabled)
             return UpdateProtocolCacheDisabledAsync();
         }
 
+        internal Task SetRequestInterceptionAsync(bool value)

```

Comment:
rename `value` to `enabled`

Reply:
I named that at first, but pw named that value at this level. It would be easier to track changes in this way.